models/profiles.yaml

```diff
@@ -27,7 +27,29 @@ models:
             select: "{{macro_datediff('{{user.Var(\"max_timestamp_bw_tracks_pages\")}}')}}"
             dependencies:
               - max_timestamp_bw_tracks_pages
-
+        # New boolean feature based on days_since_last_seen
+        - entity_var:
+            name: has_seen_recently
+            select: CASE 
+                    WHEN RAND() > 0.99 THEN null
+                    WHEN {{user.Var("days_since_last_seen")}} <= 7 
+                    THEN true
+                    ELSE false END
+            description: Whether the user has added a product to seen recently (within the last 7 days).
+        # New categorical feature based on days_since_last_seen
+        - entity_var:
+            name: last_seen_category
+            select: CASE
+                  WHEN RAND() > 0.99 THEN null
+                  WHEN {{user.Var("days_since_last_seen")}} <= 7 THEN 'recent'
+                  WHEN {{user.Var("days_since_last_seen")}} <= 30 THEN 'moderate'
+                  ELSE 'inactive' END  
+            description: Categorical variable indicating seen activity.
+      # New array type feature based on days_since_last_seen
+        - entity_var:
+            name: last_seen_days_array
+            select: CASE WHEN RAND() > 0.99 THEN null ELSE ARRAY({{user.Var("days_since_last_seen")}}) END
```

Comment:
Can you add one more array type col similar to last_seen_category -where you aggregate strings instead of integers? Keep this too - here you are aggregating ints. Change the column names to:
`int_arrays`, `string_arrays`